lib/fathead/sass/parse_functions.py

```diff
@@ -251,8 +234,8 @@ def parse_for_data(self):
                 if example:
                     abstract =  abstract + example
                 if parameter:
-                     abstract = "%s<br>%s"%(abstract, parameter)
-
+                     abstract = "%s%s"%(abstract, parameter)
+                abstract = '<div class="prog__container">' + abstract + "</div>"
```

Comment:
@VeerpalBrar we recently found out that using a <div> here is going to cause unseen problems -- can you switch it to a <section> instead?

I'm going to update the docs shortly!

Thanks 👍